webapp/app/forms/steps/lotse/personal_data_steps.py

```diff
@@ -299,6 +300,31 @@ def __init__(self, **kwargs):
             template='lotse/form_person_a.html'
         )
 
+    @classmethod
+    def get_label(cls, data=None):
+        return ngettext('form.lotse.step_person_a.label', 'form.lotse.step_person_a.label',
+                        num=get_number_of_users(data))
+
+    def render(self, data, render_info):
+        number_of_users = get_number_of_users(data)
+        render_info.step_title = ngettext('form.lotse.person-a-title', 'form.lotse.person-a-title',
+                                          num=number_of_users)
+        render_info.step_intro = ngettext('form.lotse.person-a-intro', 'form.lotse.person-a-intro',
+                                          num=number_of_users)
```

Comment:
Can we already do that in init? Because now, we're setting a different title + intro in init than in render and I guess that's confusing.

Reply:
I do not like it, too. The problem is that we need the data and so far we do not get the data in the init (we could get it, it's in the session anyway but that would be different behavior). WDYT?

Reply:
We decided to set the values to None in init and write a comment

Reply:
... or I would go back to the approach of using a conditional (maybe you can have a quick look)?

Reply:
Yes, the conditional actually really looks cleaner. 👌